ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_pooling_functions.py

```diff
@@ -173,7 +171,12 @@ def test_torch_avg_pool3d(
 ):
     input_dtype, x, kernel_size, stride, padding = dtype_x_k_s
 
-    padding = [padding[i][0] for i in range(len(padding))]
+    if len(stride) == 1:
+        stride = [stride[0]] * 3
+    if padding == "SAME":
+        padding = calculate_same_padding(kernel_size, stride, x[0].shape[2:])
+    elif padding == "VALID":
+        padding = (0, 0, 0)
```

Comment:
Same comment as in the other PR. We can probably avoid handling `"VALID"` and `"SAME"` in the tests.

Reply:
This is not something we should be filtering in the tests. The tests should cover all cases supported by the ground truth function (in this case the original torch pooling function). It's up to our frontend implementation to make sure our function supports the same cases as well.